packages/circuit-ui/hooks/useScrollLock/useScrollLock.ts

```diff
@@ -31,18 +31,20 @@ export const useScrollLock = (isLocked: boolean): void => {
   }, []);
 
   useEffect(() => {
```

Comment:
This `useEffect` should be combined with the one below, since they have the same dependencies now. This would avoid calling `restoreScroll` twice.